File: 2.sumRange/sum.cs
/* sum.cs */
// Range Sum
using System;

namespace Sum
{
  public class Program
  {
    public static long SumRange(long num)
    {
      long sum = 0L;
      for (long i = 1; i <= num; i++)
      {
        sum += i;
      }
      return sum;
    }
    public static void Main(string[] args)
    {
      string userInput;
      if (args.Length == 0)
      {
        Console.WriteLine("Enter Range");
        userInput = Console.ReadLine();
      }
      else
      {
        userInput = args[0];
      }
      long num = 0L;
      try
      {
        num = Convert.ToInt64(userInput);
      }
      catch (FormatException)
      {
        Console.WriteLine("Input Error.");
        return;
      }
      long result = SumRange(num);
      Console.WriteLine("Result: {0}", result);
    }
  }
}

File: 2.sumRange/sum.js
/* sum.js */
// Sum Range

function sumRange(num)
{
  let sum = 0n;
  for (let i = 1n; i <= num; i++)
  {
    sum += i;
  }
  return sum;
}

function show(userInput)
{
  let num = parseInt(userInput);
  if (isNaN(num))
  {
    console.log("Input Error.");
    process.exit(-1);
  }
  let result = sumRange(num);
  console.log(`Result: ${result}`);
}

if (process.argv.length <= 2)
{
  let reader = (require("readline")).createInterface({input: process.stdin, output: process.stdout});
  reader.question("Enter Range\n", function (answer) {
    show(answer);
    reader.close();
  });
}
else
{
    show(process.argv[2]);
}
